src/views/ProfileView.vue

```diff
@@ -28,8 +30,18 @@
 
   let personInfoStore: PersonInfoStore = usePersonInfoStore();
   let personStore: PersonStore = usePersonStore();
+  let authStore: AuthStore = useAuthStore();
   const personalData: Ref = ref<LabelValue[]>([]);
   const schulDaten: Ref = ref<SchulDaten[]>([]);
+  const lastPasswordChangeDate: ComputedRef<string> = computed(() => {
+    const date: Date = new Date(authStore.currentUser?.password_updated_at ?? '');
+    if (date.toString() == 'Invalid Date') return '';
+    return new Intl.DateTimeFormat('de-DE', {
+      year: 'numeric',
+      month: '2-digit',
+      day: '2-digit',
+    }).format(date);
+  });
```

Comment:
```suggestion
const lastPasswordChangeDate: ComputedRef<string> = computed(() => {
  // Hier vermeidest du dann den Coalescing operator mit einem leeren String.
  const passwordUpdatedAt = authStore.currentUser?.password_updated_at;
  if (!passwordUpdatedAt) return '';
  const date: Date = new Date(passwordUpdatedAt);
  
  // Hier auch besser, weil du dann kein String comparison machen musst mit einem hardcoded string
  if (isNaN(date.getTime())) return ''; // Better check for invalid date
  
  return new Intl.DateTimeFormat('de-DE', {
    year: 'numeric',
    month: '2-digit',
    day: '2-digit',
  }).format(date);
});

```

Reply:
Changed in [fab1f3a](https://github.com/dBildungsplattform/schulportal-client/pull/236/commits/fab1f3ad08e41e8d6ec89be11e9471783aa4d54b)